packages/fastify/src/constants.ts

```diff
@@ -0,0 +1,11 @@
+import { constants } from '@clerk/backend';
+
+export const API_URL = process.env.CLERK_API_URL || 'https://api.clerk.dev';
+export const API_VERSION = process.env.CLERK_API_VERSION || 'v1';
+export const API_KEY = process.env.CLERK_API_KEY || '';
+export const SECRET_KEY = process.env.CLERK_SECRET_KEY || '';
+export const FRONTEND_API = process.env.CLERK_FRONTEND_API || '';
```

Comment:
🔧 We should mark SECRET_KEY and FRONTEND_API key as deprecated as we do in other packages and promote the use of PUBLISHABLE_KEY and SECRET_KEY

Reply:
Do you mean we should mark `API_KEY`  and `FRONTEND_API` as deprecated ? Also after checking the codebase there are some places where we don't mark them as deprecated. This module is not a public API but an internal module. Should i mark them here as deprecated ?

Reply:
Dropping referrences and parameters for `API_KEY|apiKey` and `FRONTEND_API|frontentApi` results in errors for `clerkClient.authenticateRequest` and `clerkClient.localInterstitial`.
<img width="868" alt="Screenshot 2023-02-07 at 14 34 26" src="https://user-images.githubusercontent.com/3936408/217246079-30e57c21-fdd7-432e-a46f-7fb81528e196.png">